modules/maven-plugin-tester/pom.xml

```diff
@@ -25,14 +25,24 @@
                 <groupId>com.smartbear.readyapi</groupId>
                 <artifactId>readyapi4j-maven-plugin</artifactId>
                 <version>${project.version}</version>
-                <configuration>
-                    <username>demoUser</username>
-                    <password>demoPassword</password>
-                    <server>http://testserver.readyapi.io:8080</server>
-                </configuration>
                 <executions>
                     <execution>
-                        <id>run</id>
+                        <id>RunLocally</id>
+                        <configuration>
+                            <runXmlProjects>false</runXmlProjects>
```

Comment:
Ok, so should we remove the default value then for projects directory? or should we check if there is any project in default projects directory then fail for local execution?